### ISSUE TITLE
Issues with replication of the explore_model() population history graph figure  

### ISSUE THREAD

**Issue by archgen** [bodkan/slendr#111]
Hi Martin,

In following your slendr.net simulation tutorial my output of the Population history graph from the explore_model function gives a 'tree-like' figure rather than a more 'graph-like' figure from your webpage tutorial. Do you know if this would be due to the version of my shiny package? Or If there is a way to interact with the app to recreate the figure from your tutorial? 

Many thanks.

**Comment by bodkan** [comment]
Hello Matthew!

No, nothing is wrong with your shiny version. The graph-based rendering was giving me huge headaches for very complex models (the layout algorithm in ggraph was sometimes a bit buggy). Therefore I implemented a much simpler (although less exciting) visualization of models which could visualize Ne sizes too. Then, rather than having two ways of doing almost the same (with the graph-based viz being often buggy for crowded models), I scraped the graph-based rendering altogether. I would like to bring it back at some point because I'd like to have a way to overlay admixture graphs on a map, but that's a low priority feature right now.

Long story short, I just never got to update the image from the tutorial because it's a screenshot, not a dynamic visualization rendered via RMarkdown.

I will leave this open to remind myself to fix this.

Thanks for bringing this to my attention.

**Comment by bodkan** [comment]
Fixed by #111
